packages/devtools_app/test/common_widgets_test.dart

```diff
@@ -211,4 +211,87 @@ void main() {
       expect(findCheckboxValue(), isTrue);
     });
   });
+
+  group('AreaPaneHeader', () {
```

Comment:
I would be interested if anyone has any recommendations for these specs. I feel like the way I've grouped them makes sense, but I wonder if there is a better way to do it.